listCustomers.php

```diff
@@ -0,0 +1,61 @@
+<?php 
+include("isJson.php");
+include("greatCircleDistance.php");
+
+$intercomLatitude = "53.339428"; 
+$intercomLongitude = "-6.257664";
+$errorMessage = "";
+$listToInvite = array();
+
+$contents = @file_get_contents("customers.json");
+
+if($contents === FALSE){
+	$errorMessage = "Sorry, no such file exists";
+}else{
+	if(isJson($contents)){

```

Comment:
It's a bit strange to use this isJson check, then decode again straight afterwards. You end up having to decode twice. Maybe one function which decoded and threw an exception if there was an error?